### DISCUSSION TITLE
How to set `fixVersions` field when creating an issue and filter by that field?

### DISCUSSION THREAD

**Discussion by anatoly-scherbakov** [ankitpokhrel/jira-cli#161] [Q&A] [answered]
Is this available via some kind of customization of the application or it has to be directly supported in code to work?

**Comment by ankitpokhrel** [comment, marked as answer]
Unfortunately, this needs to be directly supported in the code to work.

There can be a lot of fields like this that an end user might be interested in. And, it might not be possible to accomodate everything. However, if there is a enough usecase for this field we can consider accepting it as a metadata. Personally, I've never used this field so I lack a bit of knowledge on its usage.

**Reply by anatoly-scherbakov** [reply]
When preferring Kanban to Scrum and thus unwilling to use Sprints, you still often want to split the timeline into manageable sections. That's why some use Versions in Jira, I suppose. Say, you have an `1.0 beta` version, you assign issues to it and manage them this way deciding which to work on for this version and which can be postponed to another version.

I do not see a clear use to `affectsVersions` field though :)

**Reply by klammbueddel** [reply]
I'd love to see fixVersions as a supported field. If we want a feature to be part of a release, we assign it to its version. The "Releases" page is quite useful to see the general progress and to "Release" a version with all tasks resolved. Making fixVersions of an issue visible and editable would help a lot. (We not ever used affectsVersions)

**Comment by ankitpokhrel** [comment]
Came here to mention that support for `fixVersions` on issue create is available since [v1.0.0-beta](https://github.com/ankitpokhrel/jira-cli/releases/tag/v1.0.0-beta).
